app/components/DosesDescriptionList/DosesDescriptionList.module.scss

```diff
@@ -0,0 +1,41 @@
+.description-definition {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.term {
+  width: 8rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.definition-primary {
+  width: 2rem;
+  text-align: right;
+}
+
+.definition-secondary {
+  flex: 1 1 0%;
+  text-align: right;
+}
+
+.check {
+  width: 1rem;
+  height: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--progress-comlete);
+  background: var(--complete-background);
+  border-radius: 99999px;
```

Comment:
What to do about this? - We could reference `./styles.scss`, which we can do by importing the stylesheet OR should the border radius be a css `var`

I don't know the answer hehe

Reply:
There's a css var for this radius value right?

Reply:
@finnclark not really, it's an `scss` var, but it hasn't been defined on the `:root` so it's not a css variable - do you think these should be?

https://github.com/theramis/aucvid/pull/55/files#diff-6181a4434ec7490436656df1775ef3f851c697d0408c56ace521630c2c7f278eR8

Reply:
oh sorry. I reckon just importing the scss then. The css vars are really only being used for colors atm

Reply:
Cool, happy with that approach. I'll tidy this up in the next PR, since we can't import the css exactly as is, will need some minor refactoring - nothing too arduous tho 💃